app/metrics.yaml

```diff
@@ -209,7 +209,8 @@ events:
             Settings, Help, Desktop Site toggle on/off, Find in Page,
             New Tab, Private Tab, Share, Report Site Issue, Back/Forward button,
             Reload Button, Quit, Reader Mode On, Reader Mode Off, Open In app,
-            Add To Top Sites, Add-ons Manager, Bookmarks, History
+            Add To Top Sites, Add-ons Manager, Bookmarks, History,
+            Set Default Browser
```

Comment:
This new event was added as part of https://github.com/mozilla-mobile/fenix/pull/18844